package.json

```diff
@@ -24,6 +24,7 @@
         "@types/feather-icons": "^4.7.0",
         "anchor-link": "^3.3.0",
         "anchor-link-browser-transport": "^3.2.0",
+        "antelope-tokens": "git+https://github.com/apporc/antelope-tokens.git",
```

Comment:
This should live under greymass or since it's just a single json file I think we can vendor it with unicove unless it's needed elsewhere

Reply:
Created + cloned: https://github.com/greymass/antelope-tokens.json

We can change the dependency to this one 👍